.github/workflows/ci.yml

```diff
@@ -185,3 +185,22 @@ jobs:
           Sincerely,
           The Miri Cronjobs Bot' \
             --user $ZULIP_BOT_EMAIL --api-key $ZULIP_API_TOKEN --site https://rust-lang.zulipchat.com
+      - name: install josh-proxy
+        run: cargo +stable install josh-proxy --git https://github.com/josh-project/josh --tag r22.12.06
+      - name: start josh-proxy
+        run: josh-proxy --local=$HOME/.cache/josh --remote=https://github.com --no-background &
+      - name: setup bot git name and email
+        run: |
+          git config --global user.name 'The Miri Conjob Bot'
+          git config --global user.email 'miri@cron.bot'
+      - name: get changes from rustc
+        run: ./miri rustc-pull
+      - name: Push changes to a branch
+        run: |
+          git switch -c "rustup$(date -u +%Y-%m)"
+          git push
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

Comment:
Looks like this can now be done without third-party actions: https://stackoverflow.com/a/73340290.

Also would be better to avoid computing the branch name twice.

Reply:
Ah yea, I just picked the first answer to that question and didn't read on :D